mobify/progressive-web-scaffold
# Clean up npm scripts.

Cleaner, more streamlined npm scripts.

## Changes
- Updated npm scripts.
- Adjusted the circle.yml config file accordingly.

## How to test-drive this PR
- Run `npm run lint` and make sure both JS and scss files are linted.
- Run `npm run test:all` and make sure all linting happens as above, plus AVA tests run.
  - Check CircleCI and make sure it does the same.
